app.json

```diff
@@ -39,6 +39,22 @@
       "description": "Algolia Places API key",
       "required": false
     },
+    "ATHENA_ACCESS_KEY_ID": {
```

Comment:
We shouldn't be adding separate access keys for Athena, we should just add the appropriate permissions to the AWS credentials that the application is already using.

Reply:
Agreed! Can you give the rc credentials for open access to Athena?

Reply:
Please open a ticket for that, preferably at https://trello.com/b/jLF7h6ZW/platform-engineering-open-learning-engineering

Reply:
I don't have access to that trello board

Reply:
Here is an open issue https://github.com/mitodl/open-discussions/issues/3318